priv/gettext/en/LC_MESSAGES/apns.po

```diff
@@ -62,8 +62,8 @@ msgid "Твой дэйт отменён"
 msgstr "Your date has been cancelled"
 
 #, elixir-format
-#: lib/t/push_notifications/apns.ex:103
-msgid "%{name} пригласил(а) тебя на свидание!"
+#: lib/t/push_notifications/apns.ex:114
+msgid "%{name} пригласил%{gender_a} тебя на свидание!"
 msgstr "You've been asked out on a date by %{name}!"
```

Comment:
Did you test that this works as expected for both genders and for both RU & EN?

Reply:
nope, it'll not work this way at all

Reply:
does it work now?

Reply:
![image](https://user-images.githubusercontent.com/16062322/142143543-1d2e507a-0edd-4a3e-8a9a-02412c56ff0b.png)
seems to work